packages/instantsearch.js/src/components/Template/__tests__/Template-test.tsx

```diff
@@ -56,6 +56,26 @@ describe('Template', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('can have Fragment as rootTagName with string template', () => {
+    const props = getProps({
+      rootTagName: 'fragment',
+      templates: { test: '<span>test</span>' },
+    });
+    const wrapper = render(<Template {...props} />);
+
+    expect(wrapper.container).toMatchSnapshot();
```

Comment:
good to have a regular snapshot as the other tests have that too and this file will eventually be simplified when Hogan is no longer there.